addon/services/infinity.js

```diff
@@ -413,7 +415,7 @@ export default Service.extend({
     @param {EmberInfinity.InfinityModel} infinityModel
    */
   _notifyInfinityModelUpdated(queryObject, infinityModel) {
-    const totalPages = get(this, '_totalPages');
+    const totalPages = get(infinityModel, '_totalPages');
```

Comment:
Not positive how this is connected to other changes.

Reply:
Unrelated but seems like an oversight.  Nice catch!

Reply:
Unrelated...I can pull it into another commit/PR if you prefer.